azure-pipelines/publishing/templates/release-docker-build-steps.yml

```diff
@@ -0,0 +1,74 @@
+# Template: Steps to build all components for a specific OS
+# Consume this steps template in one or more jobs by passing in parameter values.
+
+parameters:
+- name: targetOsArch        # example: debian9_arm32
+  type: string
+- name: imageVersion
+  type: string
+
+
+jobs:
+- job: ${{parameters.targetOsArch}}
+  steps:
+  - checkout: self
+    path: 's' # Copy to sources directory (templates do not do this automatically)
+  - template: ../../build/linux/du/templates/doclient-lite-docker-steps.yml
+    parameters:
+      targetOsArch: ${{parameters.targetOsArch}}
+      imageVersion: ${{parameters.imageVersion}}
+      config: minsizerel
+  - task: CopyFiles@2
+    inputs:
+      SourceFolder: '/tmp/build-deliveryoptimization-agent-${{parameters.targetOsArch}}/linux-minsizerel'
+      Contents: |
+          *.deb
+      TargetFolder: '/tmp/${{parameters.targetOsArch}}'
+      CleanTargetFolder: true
+    displayName: 'Copy all agent .deb file'
+
+  - template: ../../build/linux/du/templates/dosdkcpp-docker-steps.yml
+    parameters:
+      targetOsArch: ${{parameters.targetOsArch}}
+      imageVersion: ${{parameters.imageVersion}}
+      config: minsizerel
+  - task: CopyFiles@2
+    inputs:
+      SourceFolder: '/tmp/build-deliveryoptimization-sdk-${{parameters.targetOsArch}}/linux-minsizerel'
+      Contents: |
+          *.deb
+      TargetFolder: '/tmp/${{parameters.targetOsArch}}'
+      CleanTargetFolder: false
+    displayName: 'Copy sdk .deb files'
+
+  - template: ../../build/linux/du/templates/dopapt-docker-steps.yml
+    parameters:
+      targetOsArch: ${{parameters.targetOsArch}}
+      imageVersion: ${{parameters.imageVersion}}
+      config: minsizerel
+  - task: CopyFiles@2
+    inputs:
+      SourceFolder: '/tmp/build-deliveryoptimization-plugin-apt-${{parameters.targetOsArch}}/linux-minsizerel'
+      Contents: |
+          *.deb
+      TargetFolder: '/tmp/${{parameters.targetOsArch}}'
+      CleanTargetFolder: false
+    displayName: 'Copy plugin .deb file'
+  - task: DeleteFiles@1
+    inputs: 
+      SourceFolder: $(Build.ArtifactStagingDirectory)
+      Contents: |
+        **/*
+    displayName: 'Clean build folder before creating tar file folder'
+  - task: ArchiveFiles@2
+    inputs: 
+      rootFolderOrFile: /tmp/${{parameters.targetOsArch}}
+      includeRootFolder: False
+      archiveType: tar
+      archiveFile: $(build.ArtifactStagingDirectory)/${{parameters.targetOsArch}}-packages.tar
+    displayName: 'Create .tar file'
+  - task: PublishBuildArtifacts@1
+    inputs:
+      PathtoPublish: '$(Build.ArtifactStagingDirectory)'
+      ArtifactName: '${{parameters.targetOsArch}}'
```

Comment:
>${{parameters.targetOsArch}} [](http://example.com/codeflow?start=21&length=28)

Can we omit this name so that the .tar file shows up in the root folder of artifacts?

Reply:
sounds good to me

Reply:
Well, it is a required arg, so I'm going to modify publishing all .tar files to a folder called release-artifacts